provider/resources.go

```diff
@@ -1455,6 +1465,7 @@ func Provider() tfbridge.ProviderInfo {
 			"aws_fsx_lustre_file_system":  {Tok: awsResource(fsxMod, "LustreFileSystem")},
 			"aws_fsx_windows_file_system": {Tok: awsResource(fsxMod, "WindowsFileSystem")},
 			"aws_fsx_backup":              {Tok: awsResource(fsxMod, "Backup")},
+			"aws_fsx_ontap_file_system":   {Tok: awsResource(fsxMod, "OnTapFilesystem")},
```

Comment:
Should it be `OntapFileSystem` : similar to `WindowsFileSystem` above and I think "ONTAP" is a single acronym or brand name (I'm able to find `OntapVolumeConfiguration` in AWS C# SDK?

Reply:
You are correct here! I will update it - good suggestion

Reply:
Updated now @mikhailshilkov